php/libraries/LorisForm.class.inc

```diff
@@ -2209,7 +2260,7 @@ class LorisForm
      *
      * @return string the error message of the element
      */
-    function getElementError($el)
+    function getElementError(string $el): string
```

Comment:
```suggestion
    function getElementError(string $el): ?string 
```
migth be void

Reply:
I'll change the void return to return an empty string. Just FYI `?string` means null or string and doesn't cover void returns.

Reply:
@johnsaigle Thanks for the precision about `?`